servers/src/main/java/tachyon/master/MasterBase.java

```diff
@@ -32,18 +34,26 @@
 import tachyon.master.journal.JournalWriter;
 import tachyon.master.journal.ReadWriteJournal;
 import tachyon.proto.journal.Journal.JournalEntry;
+import tachyon.util.ThreadFactoryUtils;
 
 /**
  * This is the base class for all masters, and contains common functionality. Common functionality
  * mostly consists of journal operations, like initialization, journal tailing when in standby mode,
- * or journal writing when the master is the leader.
+ * or journal writing when the master is the leader. This class is not thread safe.
```

Comment:
It can still be useful to know that the methods in this class make no attempt to be thread safe